pandas/core/strings.py

```diff
@@ -2145,11 +2145,61 @@ def rindex(self, sub, start=0, end=None):
 
     _shared_docs['casemethods'] = ("""
     Convert strings in the Series/Index to %(type)s.
+
     Equivalent to :meth:`str.%(method)s`.
 
     Returns
     -------
-    converted : Series/Index of objects
+    Series/Index of objects
+
+    See Also
+    --------
+    Series.str.lower : Converts all characters to lower case.
+    Series.str.upper : Converts all characters to upper case.
+    Series.str.title : Converts first character of each word to upper case and
+        remaining to lower case.
+    Series.str.capitalize : Converts first character to upper case and
+        remaining to lower case.
+    Series.str.swapcase : Converts upper case to lower case and lower case to
+        upper case.
+
+    Examples
+    --------
+    >>> s = pd.Series(['lower', 'CAPITALS', 'this is a sentence', 'SwApCaSe'])
+    >>> s.str.lower()
```

Comment:
Could you show the original `s`, (simply with a line `>>> s` and its output). And then leave a blank line between it and the `lower` method, so the lower doesn't get grouped with the constructor and the original, but in a separate block as the rest?

Besides that, lgtm.